Umbraco-Cloud/Set-Up/Usage/index.md

```diff
@@ -0,0 +1,23 @@
+---
+versionFrom: 7.0.0
+---
+
+# Usage on your Umbraco Cloud project
+
+In the Umbraco Cloud settings tab, we recently introduced a new setting called Usage.
+
+From here you can see your current usage of Content nodes, Custom hostnames and Media storeage and the usage limitations for your Umbraco Cloud project and the plan that the project is on.
+
+![Usage on Cloud](images/Usage.png)
+
+## Usage limits
+
+On the image below you can see the Usage limitatons for the specific plans on Umbraco Cloud.
+
+![Usage limits on a starter plan](images/plan_limitations.png)
```

Comment:
Remember to double-check that your images show up on Github. It's very case sensitive 🙈 

```suggestion
![Usage limits on a starter plan](images/Plan_limitations.png)
```